PhysicsTools/PatAlgos/python/slimming/patElectronDRNCorrector_cfi.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
+
+patElectronsDRN = cms.EDProducer("PatElectronDRNCorrectionProducer",
```

Comment:
Do you just mean to use the fillDescriptions defaults for more parameters, or is there something more I should do here? I have updated it to use default values for the RecHit collections; should I add more defaults?

Reply:
I mean using `.clone` to avoid specifying the config options (and the types) that are already known.
https://github.com/cms-sw/cmssw/commit/3cb2459b31024656d4b75eb3ec7a8d5d6438dc62#diff-2d4d8dba5a6cd14974d0d42a9896706879c369b9ca833acca0f8e9f1883b1500R3

Reply:
I'm a little confused about how to do this. If I try to import `RecoEgamma.EgammaTools.PatPhotonDRNCorrectionProducer_cfi` I get a `ModuleNotFoundError`. Where is this default config produced by fillDescriptions()? How (if at all) does it interact with class templating?

Reply:
In your working area, you can look at `$CMSSW_BASE/cfipython` to find the default configs.

To ensure distinct cfi files are generated for different instances of the same class template, see this example: https://github.com/cms-sw/cmssw/blob/f6c204bfb2f92e827bac326385786fed876046e0/HeterogeneousCore/SonicTriton/test/TritonGraphModules.cc#L107-L108